test/clustertest/tests/MultipleLeaderSyncTest.cpp

```diff
@@ -0,0 +1,95 @@
+#include "../BedrockClusterTester.h"
+
+struct MultipleLeaderSyncTest : tpunit::TestFixture {
+    MultipleLeaderSyncTest()
+        : tpunit::TestFixture("MultipleLeaderSyncTest",
+                              BEFORE_CLASS(MultipleLeaderSyncTest::setup),
+                              AFTER_CLASS(MultipleLeaderSyncTest::teardown),
+                              TEST(MultipleLeaderSyncTest::test)
+                             ) { }
+
+    void setup() {}
```

Comment:
how come you aren't using setup or tear down? all other tests use these methods to create and clean up the tester and other variables, would be nice if your tests followed suit

Reply:
Tyler requested that I not use setup and teardown if they're simple, because without them, the tester gets destroyed when it goes out of scope. i'm not sure that one is obviously better. @tylerkaraszewski ?

Reply:
My view is that if you don't need separate `setup` and `teardown` functions, you can do this:

```
BedrockTester t(some params);
t.whatever();
// done.
```

But if you insist on using them then you need:
```
BedrockTester* t; // Needs to be a pointer because there's no default object for a BedrockTester

setup() {
    t = new Bedrocktester(params); // now we've dynamically allocated it so we need to explicitly clean it up.
}

test () {
    t->whatever(); // If this throws does our object get deleted (I think so)?
}

teardown() {
    delete t; // Wouldn't need this if we just made it on the stack.
}
```

Reply:
can we remove the setup/teardown functions then? or does the test library require them to exist and be no ops?

(I actually like the scoped objects better, it just doesn't match most of our tests)

Reply:
Yes, we can remove them from this test, they don't need to exist, they're optional.